docs/_tooling/decision-records/multirepo.md

```diff
@@ -0,0 +1,116 @@
+* no need to take data protection into account (everything is open source under the same license)
+* no need to take performance into account (we'll assume that the performance of the chosen solution is acceptable)
+
+## Previous Decisions:
+
+*Unfortunately those are not documented, therefore we cannot provide links to any decision records.*
+* In score different repositories are handled by bazel.
+* In score requirements and links are implemented via `sphinx-needs`.
+* In score versioning of requirement-links is handled via hashes.
+* We have two different mechanisms for versioning. Current assumption is that we'll use bazel to pull other repositories in a specific version, while we never pull different versions of the same repository. So basically, we have the "classic multi repo setup" situation.
+
+
+## Alternatives
+
+These are the identified alternatives. We'll dive into each one in detail.
+1) weblinks
+2) needservice
+3) needimport
+4) Needs-external-needs
+5) Copying
+
+### 1) needservice
+
+This is basically a manual approach to the problem. As long as any other solution works,
+that would be preferrable.
+
+### 2) weblinks
+
+We can simply link pages / needs in the other repositories by their full url.
+While we can ensure that those links work, everything beyond that will become problematic.
+Versioning might be solvable, but checking correct hashes (versioning) would be challenging.
+
+Bidirectional links are not possible. As we'd like the same approach everywhere, this is a **no-go**.
+
+### 3) needimport
+
+This is a sphinx-needs extension that allows to link to needs in other repositories.
+The other repositories do NOT need to be available at build time. Only their build output is required (needs.json).
+
+needs from the other repositories are imported, as if they were local.
+All structure is lost! All surrounding text, images etc are lost. Only the needs themselfes are imported. This is a **no-go**.
+
+
+### 4) Needs-external-needs (+ intersphinx)
+
+This is a sphinx-needs extension that allows to link to needs in other repositories.
+The other repositories do NOT need to be available at build time. Only their build output is required (needs.json).
+
+For bidirectional linking:
+* Build all repositories once, so we have needs.json
+* Exchange needs.json
+* Build all repositories again
+
+As is already evident in the requirements, we need multiple versions of the score repository website. One per integration. Since the pure score repository website shall not contain links to the other repositories, except when used in the context of an integration.
+
+Pro:
+* Bidirectional links are possible
+* Good performance (no need to build all repositories at once)
+Con:
+* needs.json and the entire website needs to be hosted in multiple versions.
+* bidirectional links affect the "core score" repository, which is not desired.
```

Comment:
I don't understand this point by 100%: What exactly is affected in the "core score" repository?
From my understanding changes in the repository itself are not needed, just the final documentation gets incoming links from module-repos/docs.

Reply:
Yes, I was talking about the website, not the repository. Fixed:
> bidirectional links affect the score-website, which is not desired

Reply:
Resolving, as the text is completely rewrtten.